packages/lib/OgImages.tsx

```diff
@@ -147,15 +160,7 @@ export const Meeting = ({ title, users = [], profile }: MeetingImageProps) => {
             </div>
           )}
           <div tw="flex flex-row">
-            {avatars.slice(0, 3).map((avatar) => (
```

Comment:
This can throw an error (e.g., `Can't load image http://localhost:3000/client-onboarding/avatar.png`), hence screwing up the entire preview image. Using `Image` component from next has error handling.